README.md

```diff
@@ -40,6 +40,7 @@ To run the app, follow these steps.
   gulp watch
   ```
 7. Browse to [http://localhost:9000](http://localhost:9000) to see the app. You can make changes in the code found under `src` and the browser should auto-refresh itself as you save files.
+> The Skeleton App uses [BrowserSync](http://www.browsersync.io/) for automated page refreshes on code/markup changes concurrently accross multiple browsers. If you prefer to disable the mirroring feature set the [ghostMode option](http://www.browsersync.io/docs/options/#option-ghostMode) to false
```

Comment:
The reason for it is to enable faster cross-browser testing as you said. Especially when working with Mobile devices, because it saves you tons of time not having to type inputs on the small screen :)

Didn't include too much information as I don't want to spam the README. Going into detail would make it explode pretty soon :) If you feel the need though just create a Fork off of this PR and add your changes, so we can see the result and have @EisenbergEffect decide what or if anything to merge.